script/openqa-bootstrap

```diff
@@ -102,6 +102,18 @@ key = ${API_KEY}
 secret = ${API_SECRET}
 EOF
 
+# configure worker@1 to use x86_64 backends
+cat >> /etc/openqa/workers.ini <<EOF
+[1]
+WORKER_CLASS = qemu_x86_64_staging,qemu_x86_64
+EOF
+
 
 # start worker
 systemctl enable --now openqa-worker@1.service
+
+# clone jobs if jobId is given. Possible to pass extra arguments as well.
+# ex: openqa-bootstrap -from openqa.opensuse.org 12345 SCHEDULE=tests/boot/boot_to_desktop,tests/x11/kontact
+if [ $# -ne 0 ]; then
+	/var/lib/openqa/script/openqa-clone-job "$@"
```

Comment:
Just the application from path, w/o any directory prefix, please

Reply:
done